integration-cli/docker_cli_daemon_test.go

```diff
@@ -324,6 +324,90 @@ func (s *DockerDaemonSuite) TestDaemonLogLevelWrong(c *check.C) {
 	c.Assert(s.d.Start("--log-level=bogus"), check.NotNil, check.Commentf("Daemon shouldn't start with wrong log level"))
 }
 
+func (s *DockerSuite) TestDaemonStartWithBackwardCompatibility(c *check.C) {
```

Comment:
No because we need to manually handle the daemon object. We run NewDaemon in the loop, and then need to set GlobalFlags for the invalid part